sklearn/metrics/_classification.py

```diff
@@ -2494,6 +2494,7 @@ def hinge_loss(y_true, pred_decision, *, labels=None, sample_weight=None):
     Returns
     -------
     loss : float
+        Hinge loss value.
```

Comment:
Small nit:

```suggestion
        Average hinge loss.
```

To be consistent with the docstring above:

https://github.com/scikit-learn/scikit-learn/blob/9c12fb0c36eb49de91971238fb9376c664cbe1f7/sklearn/metrics/_classification.py#L2463